sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusErrorSource.java

```diff
@@ -36,4 +36,12 @@
     /** Error while session is closed.*/
     public static final ServiceBusErrorSource CLOSE_SESSION = fromString("CLOSE_SESSION",
         ServiceBusErrorSource.class);
+
+    /** Error while sending a message.*/
+    public static final ServiceBusErrorSource SENDING = fromString("SENDING",
+            ServiceBusErrorSource.class);
+
+    /** Error while trying to do an operation on the management link. */
+    public static final ServiceBusErrorSource MANAGEMENT = fromString("MANAGEMENT",
+            ServiceBusErrorSource.class);
```

Comment:
Assuming that the Error Source will be restricted to the processor only as per https://github.com/Azure/azure-sdk-for-java/pull/17601/files#r524813422, we dont need the above additions right?

Reply:
I'm adding them in since we do throw this exception type from places other than the processor.

However, I am making the errorSource field package-private now.